third_party/move/move-compiler-v2/src/pipeline/control_flow_graph_simplifier.rs

```diff
@@ -469,6 +496,30 @@ impl ControlFlowGraphCodeGenerator {
                         .collect::<BTreeSet<_>>()
```

Comment:
In combination with the previous comment, if successors, predecessors, and its reverse map, all are sets instead of vecs to begin with (which is the logical representation, because we do not have a multi-graph), we can use iter::eq.